staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -85,7 +85,7 @@ func WithLogging(handler http.Handler, pred StacktracePred) http.Handler {
 		rl := newLogged(req, w).StacktraceWhen(pred)
 		req = req.WithContext(context.WithValue(ctx, respLoggerContextKey, rl))
 
-		defer rl.Log()
+		defer klog.V(3).InfoS("Received HTTP request", rl.LogArgs()...)
```

Comment:
```suggestion
		defer func(){klog.V(3).InfoS("Received HTTP request", rl.LogArgs()...)}()
```
We need to put it in function as defer arguments are evaluated immidietly.
Not the best but without InfoSDepth we cannot preserve same call site

Reply:
Okay